cypress/integration/Errors.feature

```diff
@@ -0,0 +1,26 @@
+@Errors
+Feature: Errors
```

Comment:
Good idea. This could potentially allow us to just run accessibility tests if we wanted.

Reply:
IMHO we should hopefully not need such a seperate feature file for accessibility. The idea was a11y gets tested all along, as other tests run. The tests in this feature file you ought to get for free anyway, since a11y will be tested when those things happen. Could do a call to sync ideas? :)

Reply:
I've mixed a11y tests in with other page tests. @nickdavis2001 has a point in that we should probably just treat accessibility as a baked-in feature of the app and not give it special treatment.

Reply:
> IMHO we should hopefully not need such a seperate feature file for accessibility. 

It's not really being separated into its own file if we apply a tag. We're re-creating/creating, all of these tests with the same level of organization as we previously had...which was none. Can we point out which tests cover; user behaviour, integration/3rd party integration or smoke test. Without having to delve into the code?

Reply:
@Raffers I don't think we need additional tags right now, as we're ignoring them anyway. I like the idea of a set of tags for smoke tests, for example, so we have a quick set of tests we can run to check things are generally OK when doing development work. Other categories might be harder to decide. But deciding what should be in each set is probably an exercise in its own right, so I'd suggest raising a ticket for that specifically.

Reply:
It was more of a case for readability as well, even though we're not actually doing anything with tags on CI. But it would be nice to get some context for what a particular test is trying to achieve based on its tags. I just think it's an important distinction if we're planning on having all test types, in the same set of files.

But yer, the categorization of these can be confusing & we'd need to agree on what parts of the system should it be touching/behavior expected before applying certain tags.

Reply:
Are we happy to resolve this for now and raise separately?